en/components/expansion_panel.md

```diff
@@ -622,6 +620,84 @@ You can see the results below:
+
+### Import theme
+First, in order for us to use the functions exposed by the theme engine, we need to import the `index` file in our style file: 
+
+```scss
+// in styles.scss
+@import '~igniteui-angular/lib/core/styles/themes/index';
+```
+
+### Define palette & colors
+After we've imported the `index` file we can go ahead and use the [`igx-color`]({environment:sassApiUrl}/index.html#function-igx-color) function to define some color variables we would like to use in our custom theme. We are going to use these for our custom `igx-expansion-panel` styling in conjunction with our own color [palette](themes/palette.md) where we can specify our main colors to be used by the component.
+Fist define a custom palette and pass our main colors:
+```scss
+// in styles.scss
+$my-primary-color:#72da67;
+$my-secondary-color: #7344df81;
+$my-info-color:rgba(0, 0, 0, 0.54)
+
+$my-color-palette: igx-palette(
+    $primary: $my-primary-color,
+    $secondary: $my-secondary-color,
+    $info: $my-info-color
+);
+```
+
+In order to see our custom palette and colors applied, we need to pass these to a theme function.
+So in one bold move we will [`create a custom theme`](themes/component-themes.md#creating-themes) and pass our colors to a number of predefined `igx-expansion-panel-theme` parameters . Let's say we have decided modifying these specific parameters will be more than sufficient to make our component look the way we like. It is really convenient to take use of the previously created [palette](themes/palette.md) and base our new colors on the colors defined.
+```scss
+// in styles.scss
+$custom-panel-theme: igx-expansion-panel-theme(
+    $palette: $my-color-palette,
+    $header-background: igx-color($my-color-palette, "primary", 200),
+    $header-focus-background: igx-color($my-color-palette, "primary", 300),
+    $header-title-color: igx-color($my-color-palette, "info"),
+    $header-icon-color: igx-color($my-color-palette, "info"),
+    $body-background: igx-color($my-color-palette, "secondary", 100),
+    $border-radius: 0.35em
+);
+```
+
+### Applying
+All that's left is to properly scope our newly created theme.
+
+#### Globally
+In case you want this newly created `igx-expansion-panel theme` to be applied [`globally`](themes/component-themes.md#creating-themes) in your app, all that is needed is to include the theme in your app root style file:
+```scss
+// in styles.scss
+// Pass our custom-panel-theme to `igx-expansion-panel` mixin.
+@include igx-expansion-panel($custom-panel-theme);
+```
+#### Scoped
+There may be a case where you want a particular `igx-expansion-panel` be styled differently than the others in the app. This will require to use angular specific pseudo-class selectors like `:host`, `::ng-deep`, etc. Additionally all of the above steps need to be moved from `styles.scss` to the `component.scss` file.
```

Comment:
```suggestion
There may be a case where you want a particular `igx-expansion-panel` to be styled differently than the others in the app. This will require to use angular specific pseudo-class selectors like `:host`, `::ng-deep`, etc. Additionally all of the above steps need to be moved from `styles.scss` to the `component.scss` file.
```